sonic-utilities-tests/sku_create_test.py

```diff
@@ -51,30 +69,197 @@ def are_file_contents_same(self,fname1,fname2):
 
         #Test case execution without stdout
         result = subprocess.check_output(my_command,stderr=subprocess.STDOUT,shell=True)
         print result
 
         #Check if the Output file exists
-        if (os.path.exists(output_file_path)):
-            print("Output file: ",output_file_path,"exists. SUCCESS!")
+        if (os.path.exists(output_xml_file_path)):
+            print("Output file: ",output_xml_file_path, "exists. SUCCESS!")
         else:
-            pytest.fail("Output file: {} does not exist. FAILURE!".format(output_file_path))
+            pytest.fail("Output file: {} does not exist. FAILURE!".format(output_xml_file_path))
 
         #Check if the Output file and the model file have same contents
-        if self.are_file_contents_same(output_file_path,model_file_path) == True:
-            print("Output file: ",output_file_path," and model file: ",model_file_path,"contents are same. SUCCESS!")
+        if self.are_file_contents_same(output_xml_file_path, model_xml_file_path) == True:
+            print("Output file: ",output_xml_file_path, " and model file: ",model_xml_file_path, "contents are same. SUCCESS!")
         else:
-            pytest.fail("Output file: {} and model file: {} contents are not same. FAILURE!".format(output_file_path,model_file_path))
+            pytest.fail("Output file: {} and model file: {} contents are not same. FAILURE!".format(output_xml_file_path, model_xml_file_path))
+
+    def test_sku_from_minigraph_file(self):
+        if (os.path.exists(output_minigraph_dir_path)):
+            shutil.rmtree(output_minigraph_dir_path)
+
+        my_command = sku_create_script + " -m "  + minigraph_file  + " -d " + minigraph_input_path
+
+        #Test case execution without stdout
+        result = subprocess.check_output(my_command,stderr=subprocess.STDOUT,shell=True)
+        print result
+
+        #Check if the Output file exists
+        if (os.path.exists(output_minigraph_file_path)):
+            print("Output file: ",output_minigraph_file_path, "exists. SUCCESS!")
+        else:
+            pytest.fail("Output file: {} does not exist. FAILURE!".format(output_minigraph_file_path))
+
+        #Check if the Output file and the model file have same contents
+        if self.are_file_contents_same(output_minigraph_file_path, model_minigraph_file_path) == True:
+            print("Output file: ",output_minigraph_file_path, " and model file: ",model_minigraph_file_path, "contents are same. SUCCESS!")
+        else:
+            pytest.fail("Output file: {} and model file: {} contents are not same. FAILURE!".format(output_minigraph_file_path, model_minigraph_file_path))
+
+    def test_sku_from_config_db_file(self):
+        if (os.path.exists(output_config_db_dir_path)):
+            shutil.rmtree(output_config_db_dir_path)
+
+        my_command = sku_create_script + " -j "  + config_db_file  + " -d " + config_db_input_path
+
+        #Test case execution without stdout
```

Comment:
Per PEP8, add space after `#`. Do this throughout the file.

Reply:
addressed in next commit